test/parse_test.exs

```diff
@@ -0,0 +1,25 @@
+defmodule ParseTest do
+  use ExUnit.Case
+
+  @urls [
+    "http://onigumo.local/hello.html",
+    "http://onigumo.local/bye.html"
+  ]
+
+  test("Parsing values of href attributes in html links") do
+    document = ~s("<!doctype html>
+      <html>
+      <body>
+        <section id="content">
+          <p class="headline">Floki</p>
+          <a href="http://onigumo.local/hello.html">Hello</a>
+          <a href="http://onigumo.local/bye.html">Bye</a>
+          <span data-model="user">onigumo</span>
+        </section>
+      </body>
+      </html>")
```

Comment:
Why are URLs in a module constant but the document in a function variable? I can’t tell which is better, but I feel like having it consistent would be better. I am personally more inclined to module constants. We already did so in the Hash [test](https://github.com/nappex/onigumo/blob/links-parser/test/hash_test.exs).